### DISCUSSION TITLE
attribute selector should not escape `/`

### DISCUSSION THREAD

**Discussion by lifeiscontent** [tailwindlabs/tailwindcss#10352] [Help]
trying to figure out how to show an external icon in scenarios where the href does not start with `/`

https://play.tailwindcss.com/sRVqoiJfLV

it appears tailwind is escaping the `/` as `//` not sure how to solve this.

**Comment by aenial51** [comment]
very informative , currently not having proper solution but I assure you very soon will update you please keep in touch .

[www.benefitscal.org
](https://www.benefitscal.ltd/)

**Comment by wongjn** [comment]
Seems like Tailwind has some trouble matching it in an arbitrary selector, but it works if you refactor the variant into a plugin: https://play.tailwindcss.com/A1PnGTVqLg?file=config
